assets/js/romo/base.js

```diff
@@ -458,16 +458,7 @@ Romo.prototype.initAfterHtml = function(elem, htmlString) {
 
 // events
 
-Romo.prototype.on = function(elem, eventName, fn) {
-  var elemString = Object.prototype.toString.call(elem);
-  if (
-    elemString === '[object NodeList]'       ||
-    elemString === '[object HTMLCollection]' ||
-    Array.isArray(elem)
-  ) {
-    throw new Error('Can only bind events on individual elems, not collections.');
-  }
-
+Romo.prototype.on = function(elems, eventName, fn) {
   var proxyFn = function(e) {
     var result = fn.apply(elem, e.detail === undefined ? [e] : [e].concat(e.detail));
```

Comment:
@jcredding did you test this at all?  I don't think this works b/c `elem` is not defined here.  Not sure what you wanna do about that.  I missed this earlier too.

Reply:
@jcredding that is passing in `elem` as the value of `this` in the function (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply).  Maybe we don't do this?  I don't think I even rely on `this` in event callbacks without using `Romo.proxy`.